oecologia-australis.csl

```diff
@@ -61,21 +61,17 @@
     <choose>
       <if type="article-journal chapter paper-conference entry-dictionary entry-encyclopedia" match="none">
         <group delimiter=", ">
-          <names variable="container-author" delimiter=", ">
-            <name and="symbol" initialize-with=". " delimiter=", "/>
-            <label form="short" prefix=", " text-case="title"/>
-          </names>
-          <names variable="editor translator" delimiter=", ">
-            <name and="symbol" initialize-with=". " delimiter=", "/>
-            <label form="short" prefix=", " text-case="title"/>
+          <names variable="editor translator" delimiter=", " suffix=" ">
+            <name delimiter=" " suffix="," and="symbol" initialize-with=". "/>
+            <label form="short" text-case="title" prefix=" "/>
           </names>
         </group>
       </if>
     </choose>
   </macro>
   <macro name="author">
     <names variable="author">
-      <name and="symbol" delimiter-precedes-last="always" initialize-with=". " name-as-sort-order="all"/>
+      <name and="symbol" delimiter-precedes-last="always" et-al-min="1" et-al-use-first="0" initialize-with=". " name-as-sort-order="all"/>
```

Comment:
> I need that all author are included on all types of references

"et-al" settings are a bit tricky, because you can specify them in different places in CSL styles. To always print all authors in the bibliography, delete `et-al-min="1" et-al-use-first="0"` here in the "author" macro, and also `et-al-min="8" et-al-use-first="6" et-al-use-last="true"` at the start of the bibliography section (in the line `<bibliography hanging-indent="true" et-al-min="8" et-al-use-first="6" et-al-use-last="true" entry-spacing="0" line-spacing="2">`).